src/server/bg_services/server_monitor.js

```diff
@@ -69,61 +69,14 @@ async function run_monitors() {
 }
 
 function _check_dns_and_phonehome() {
-    dbg.log2('_check_dns_and_phonehome');
-    return phone_home_utils.verify_connection_to_phonehome()
-        .then(res => {
-            switch (res) {
-                case "CONNECTED":
-                    monitoring_status.dns_status = "OPERATIONAL";
-                    monitoring_status.ph_status = {
-                        status: "OPERATIONAL",
-                        test_time: moment().unix()
-                    };
-                    break;
-                case "MALFORMED_RESPONSE":
-                    monitoring_status.dns_status = "OPERATIONAL";
-                    monitoring_status.ph_status = {
-                        status: "FAULTY",
-                        test_time: moment().unix()
-                    };
-                    break;
-                case "CANNOT_CONNECT_PHONEHOME_SERVER":
-                    monitoring_status.dns_status = "OPERATIONAL";
-                    monitoring_status.ph_status = {
-                        status: "UNREACHABLE",
-                        test_time: moment().unix()
-                    };
-                    Dispatcher.instance().alert('MAJOR',
-                        system_store.data.systems[0]._id,
-                        `Phone home server could not be reached`,
-                        Dispatcher.rules.once_daily);
-                    break;
-                case "CANNOT_CONNECT_INTERNET":
-                    monitoring_status.internet_connectivity = "FAULTY";
-                    Dispatcher.instance().alert('MAJOR',
-                        system_store.data.systems[0]._id,
-                        `Phone home server could not be reached, phone home connectivity is used for proactive
-                         support product statistics analysis. Check internet connection`,
-                        Dispatcher.rules.once_daily);
-                    break;
-                case "CANNOT_RESOLVE_PHONEHOME_NAME":
-                    monitoring_status.dns_status = "FAULTY";
-                    Dispatcher.instance().alert('MAJOR',
-                        system_store.data.systems[0]._id,
-                        `DNS server/s cannot resolve Phone home server name`,
-                        Dispatcher.rules.once_daily);
-                    break;
-                case "CANNOT_REACH_DNS_SERVER":
-                    monitoring_status.dns_status = "UNREACHABLE";
-                    break;
-                default:
-                    break;
-            }
-            if (_.isEmpty(server_conf.dns_servers)) {
-                delete monitoring_status.dns_status;
-            }
-        })
-        .catch(err => dbg.warn('Error when trying to check dns and phonehome status.', err.stack || err));
+    if (!_.isEmpty(server_conf.dns_servers)) {
```

Comment:
don't understand why we are still returning those... sound like it will never change. maybe we don't need check_dns_and_phonehome at all...

Reply:
I think we don't I just wanted to remove the ping we do to the old ph server. But agree we need to clean up the schema and API especially around the server HB area